### PR TITLE
Fix name in trainer card

### PR DESCRIPTION
**Pull request by whalemer**
[02:42:41] peach: name never starts with a rank symbol on initialization
[02:42:50] peach: and it never starts with a rank symbol in userdetails
[02:43:10] peach: so i think you can just delete the line i linked as well as L2467 and be fine

### CODE REVIEW

File: js/client.js [hunk 1]
```diff
@@ -2497,7 +2496,6 @@ function toId() {
 					globalgroup = "Global " + globalgroup;
 				}
 			}
-			if (group || name.charAt(0) === ' ') name = name.substr(1);
```
Comment (Zarel) [inline js/client.js]: I would replace this with an ID detector.

**Comment by Zarel** [comment]
> [02:42:41] peach: name never starts with a rank symbol on initialization

I don't know if I believe this. I'm pretty sure it does.

**Comment by Zarel** [comment]
Oh, huh, apparently it was separated into `roomgroup` at some point. I suppose that's fine by me.

**Comment by whalemer** [comment]
As far as i can tell, the only time that'd be meaningful would be for the
moment after I did `/user ~Zarel` before the server responded, so I'm not
sure what it really achieves.

**Pull request merged**